File: TravellerCreator/ClientApp/src/app/sector-creator/models/star-frontiers/star-frontiers-subsector.ts
﻿import {ISubsector} from "../interfaces/subsector";
import {Coordinates} from "../other/coordinates";
import {StarFrontiersHex} from "./star-frontiers-hex";

export class StarFrontiersSubsector implements ISubsector {
  coordinates: Coordinates;
  hexes: StarFrontiersHex[];
}

File: TravellerCreatorModels/StarFrontiers/StarFrontiersSubsector.cs
using TravellerCreatorModels.Interfaces;
using TravellerCreatorModels.Other;

namespace TravellerCreatorModels.StarFrontiers;

public class StarFrontiersSubsector : ISubsector
{
    public Coordinates? Coordinates { get; set; }
    public List<IHex> Hexes { get; set; } = new();
}